README.md

```diff
@@ -24,6 +24,37 @@ The NuGet Docs are the ideal place to start if you are new to NuGet. They are ca
 
 The NuGet Blog is where we announce new features, write engineering blog posts, demonstrate proof-of-concepts and features under development.
 
+## Contributing
+
+There are many ways in which you can participate in the project, for example:
+
+* [Submit bugs and feature requests, and help us verify as they are checked in](https://github.com/NuGet/Home/wiki/Submitting-Bugs-and-Suggestions)
+* [Review source code changes](https://github.com/NuGet/Home/pulls)
```

Comment:
This suggests there's source code in this repo, but it's not really true. 
This is our designs repo. Not sure how specific we want to be here (given the whole, many repos thing :) )

Reply:
Then again, this whole page does feel like very client focused, even before these changes, so maybe just linking the client repo is good enough.

Reply:
@nkolev92 I think you bring up a great point. This repo serves as a "catch-all", yet we also point users to other specific repos that do have source code & where issues eventually get routed.

Perhaps we need to be more specific on reviewing source code changes for [each repo](https://github.com/NuGet/Home#repos-and-projects). Perhaps we can change `Review source code changes` in this specific instance to `Review new NuGet proposals` instead?

Reply:
I think that works. 
Contribute to NuGet contains info about individual repos, https://github.com/NuGet/Home/wiki/Contribute-to-NuGet.